config/locales-glitch/es.yml

```diff
@@ -0,0 +1,33 @@
+---
+es:
+  admin:
+    dashboard:
+      keybase: Integración con keybase
+    settings:
+      enable_keybase:
+        desc_html: Permite a tus usuarixs comprobar su identidad por medio de keybase
+        title: Habilitar la integración con keybase
+      outgoing_spoilers:
+        desc_html: Cuando los toots federen, agrega esta etiqueta de contenido a los toots que no tengan. Es útil si tu servidor se especializa en contenido que otros servidores desearían tener con una advertencia de contenido. Los medios también se marcarán como sensibles.
+        title: Advertencia de contenido para los toots salientes
+      hide_followers_count:
+        desc_html: No mostrar el conteo de seguidorxs en perfiles de usuarix
+        title: Ocultar conteo de seguidorxs
+      show_reblogs_in_public_timelines:
+        desc_html: Mostrar retoots públicos en las línea de tiempo local y pública.
+        title: Mostrar retoots en líneas de tiempo públicas
+      show_replies_in_public_timelines:
+        desc_html: Además de auto-respuestas públicas (hilos), mostrar respuestas públicas en las línea de tiempo local y pública.
+        title: Mostrar respuestas en líneas de tiempo públicas
+  generic:
+    use_this: Usar
+  settings:
+    flavours: Ediciones
+  otp_authentication:
+    code_hint: Ingresa el código generado por tu aplicación de autenticación para confirmar
+    description_html: Si habilitas la <strong>autenticación de dos factores</strong> usando una aplicación de autenticación, se requerirá que tengas tu dispositivo a la mano al iniciar sesión, este generará un código numérico o token para que lo ingreses.
+    enable: Habilitar
+    instructions_html: <strong>Escanea este código QR en Authy, Google Authenticator o en otra aplicación TOTP en tu dispositivo</strong>. A partir de ahora, esa aplicación generará un código numérico o token para que lo ingreses.
+    manual_instructions: "Si no puedes escanear el código QR y necesitas ingresarlo manualmente, aquí está el secreto en texto simple:"
+    setup: Configurar
+    wrong_code: ¡El código que ingresaste no es válido! ¿La hora del servidor y del dispositivo son correctas?
```

Comment:
Those aren't glitch-soc additions, those are from upstream and should be translated there.

Reply:
I agree, in upstream it has been already been translated.
Over there `otp_authentication` is called `two_factor_authentication`. https://github.com/tootsuite/mastodon/blob/054f4af603b2edc44fa0bf82eb9373c691d1d9a3/config/locales/es.yml#L1264
Changing the variable name in `config/locales/es.yml` causes CI checks to fail, that's why I put that there, since Glitchsoc variable names differ from upstream.
I apologize if it is incorrect.

Reply:
Hmmm... there really shouldn't be any difference between glitch-soc and upstream regarding this?

Reply:
I'm sorry, maybe I had an old version or something, I'll fix it.

Reply:
There's something confusing me, to make sure my changes weren't the problem and to be extra careful, I cloned glitch again and copied public/system and .env.production (renamed directory to live, cleared caches, etc) and still, the content after the title in `https://<myinstance>/settings/otp_authentication` is shown in english.

I'm not that familiar with glitch and masto code to know what's happening, I think I changed the variable name in the first place when I noticed that in my glich instance, the link to 2fa is `/settings/otp_authentication` and in other Mastodon v3.2.0 it is `/settings/two_factor_authentication`.

Reply:
The title for `https://<myinstance>/settings/otp_authentication` is `settings.two_factor_authentication`.

The difference isn't between glitch-soc and upstream but between 3.2.0 and the current development version.

Reply:
The requested changes have been made :)